create-snowpack-app/app-template-preact-typescript/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@snowpack/app-template-preact-typescript",
+  "version": "1.10.2",
+  "license": "MIT",
+  "homepage": "https://github.com/pikapkg/snowpack/tree/master/create-snowpack-app/app-template-preact-typescript#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pikapkg/snowpack.git",
+    "directory": "create-snowpack-app/app-template-preact-typescript"
+  },
+  "keywords": [
+    "csa-template"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "start": "snowpack dev",
+    "build": "snowpack build",
+    "format": "prettier --write \"src/**/*.{js,jsx}\"",
+    "lint": "prettier --check \"src/**/*.{js,jsx}\"",
+    "test": "jest"
+  },
+  "dependencies": {
+    "preact": "^10.5.4"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.11.6",
+    "@babel/preset-react": "^7.10.4",
+    "@babel/preset-typescript": "^7.10.4",
+    "@prefresh/snowpack": "^2.0.1",
+    "@snowpack/app-scripts-preact": "^1.9.3",
```

Comment:
Maybe it would be better to remove ```@snowpack/app-scripts-preact``` from dependencies and move jest.config to the template?

Reply:
I think we’re heading in that direction, but the Jest configuration is just annoying enough where I think it’s better to leave `app-scripts-preact` as-is for now. However, we may be able to ditch it when #1197 gets a little further! But for this PR I think this is fine for now.